packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -111,6 +97,15 @@ export const PieChart: FC<Props> = ({
     );
   };
 
+  const fieldDisplayValues = getFieldDisplayValues({
+    fieldConfig,
+    reduceOptions: reduceOptions,
+    data: data,
+    theme: useTheme(),
```

Comment:
Hooks should be used as top most calls in a component  (https://ru.react.js.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)